### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by boaz-codota**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    "airbnb-base",
```
Comment (dimacodota) [inline .eslintrc.js:7]: airbnb-base rulzz 🚀

**Comment by boaz-codota** [comment]
WIP: Do not merge

**Review by dimacodota — APPROVED** [review]
LGTM

**Review by dimacodota — APPROVED** [review]
LGTM

**Pull request merged**